CC/Sounder/include/config.h

```diff
@@ -178,10 +190,14 @@ class Config {
     {
         return this->pilot_sym_;
     };
-    inline std::vector<std::vector<int>>& pilot_sc(void)
+    inline std::vector<std::complex<float>>& pilot_sc(void)
```

Comment:
Not sure if it is ok to return a const reference, depends on usage but if it is not necessary for the caller to modify the values, make the functions return a const reference (maybe add a getter and setter function).